source/install/desktop-app-install.rst

```diff
@@ -231,19 +231,21 @@ For further assistance, review the `Troubleshooting forum <https://forum.matterm
 Reporting Issues
 ----------------
 
-When reporting bugs found in the Mattermost desktop app, it is helpful to include the contents of the Developer Tools Console along with `the information on this page <https://docs.mattermost.com/process/support.html#general-questions-for-any-issues>`__. To access the Developer Tools Console, follow these instructions:
+When reporting bugs found in the Mattermost Desktop App, it's helpful to include the contents of the Developer Tools Console along with `the information on this page <https://docs.mattermost.com/process/support.html#general-questions-for-any-issues>`__. 
 
-1. In the menu bar, go to **View > Toggle Developer Tools**.
+To access the Developer Tools Console, follow these instructions:
+
+1. In the menu bar, go to **View > Developer Tools for Current Tab**.
 2. Select the **Console** tab.
-3. Right-click the log window and select **Save As**.
-4. Save the file and then send it along with a description of your issue.
-5. Go to **View > Toggle Developer Tools** to disable the Developer Tools.
+3. Right-click the log entry, then select **Save As**.
+4. Save the file, then send it along with a description of your issue.
+5. Close the console to disable the Developer Tools.
 
 You can open an additional set of developer tools for each server you have added to the desktop app. The tools can be opened by pasting this command in the Developer Tools Console you opened with the steps described above:
 
 ``document.getElementsByTagName("webview")[0].openDevTools();`` 
 
-Note that if you have more than one server added to the desktop client, you need to change the 0 to the number corresponding to the server you want to open in the Developer Tools Console, starting with 0 from the left.
+When you have more than one server added to the Desktop App, you need to change the ``0`` in the command above to the number corresponding to the server you want to open in the Developer Tools Console, starting with ``0`` on the left.
```

Comment:
This step isn't really needed anymore since the dev console can simply be opened through the desktop app in the "View" dropdown.